spec/avram/queryable_spec.cr

```diff
@@ -1247,7 +1247,7 @@ describe Avram::Queryable do
       UserQuery.new.select_count.should eq 10
       # NOTE: we don't test rows_affected here because this isn't
       # available with a truncate statement
-      UserQuery.truncate
+      UserQuery.truncate(cascade: true)
```

Comment:
This had to change because I added the `has_many transactions` to User, and now it was necessary.